clang/lib/Sema/SemaBounds.cpp

```diff
@@ -841,13 +845,13 @@ namespace {
 
         OS << "{\n";
         for (auto I = OrderedDecls.begin(); I != OrderedDecls.end(); ++I) {
-          DeclaratorDecl *Variable = *I;
-          if (!UC[Variable])
+          const VarDecl *Variable = *I;
+          if (!Context[Variable])
```

Comment:
I think this would insert the Variable in Context if it does not exist.

If you only want to test existence of key in a map please use this:
`if (Context.count(Variable))`

If you want to test existence *and* use the value, then use this:
```
auto it = Context.find(Variable);
if (it != Context.end()) {
  // use *it
}
```